src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
+const HEALTH_DECREASE_COUNT = 50;
+const DEFAULT_HEALTH = 100;
+
 class Animal {
   // write your code here
+  constructor(name, health = DEFAULT_HEALTH) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
   // write your code here
+  constructor(name, health = DEFAULT_HEALTH) {
+    super(name, health);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
   // write your code here
+  bite(bitedAnimal) {
```

Comment:
You can say a more universal name of the argument
```suggestion
  bite(victim) {
```